languages/python/exercises/concept/string-methods/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
+
+You have four tasks to clean up and modify strings.
+
+## 1. Capitalize the title of the paper
+
+Any good paper needs a properly formatted title. Implement the function `capitalize_title()` which takes in the parameter `title` and capitalizes the first letter of each word of the title. This function will return a `string`.
+
+```python
+capitalize_title("my hobbies")
+"My Hobbies"
+```
+
+## 2. Check if each sentence ends with a period
+
+You want to make sure that the punctuation in the paper is perfect. Implement the function `check_sentence_ending()` that takes in the parameter `sentence`. This function should return a `boolean`.
```

Comment:
Hmm... I'm not sure if "sentence" should be wrapped in backticks or not.

@BethanyG, what convention are we following with function parameter names like this?

```suggestion
You want to make sure that the punctuation in the paper is perfect. Implement the function `check_sentence_ending()` that takes in the parameter `sentence`. This function should return a `bool`.
```